node.gyp

```diff
@@ -1225,6 +1225,16 @@
       ],
 
       'conditions': [
+        [ 'node_use_openssl=="true"', {
```

Comment:
Just realized that these are necessary for the headers (esp. enums) to match in mkcodecache and node_mksnapshot binaries